src/Config/CS.props

```diff
@@ -10,9 +10,9 @@
 	<REVIT_VERSION>Revit_$(RevitVersionNumber)</REVIT_VERSION>
 	<OutputPath Condition=" '$(OutputPath)' == '' ">$(SolutionDir)..\bin\$(Platform)\$(Configuration)\$(REVIT_VERSION)</OutputPath>
     <NunitPath Condition=" '$(NunitPath)' == '' ">$(SolutionDir)..\extern\NUnit</NunitPath>
-	<DYNAMOAPI  Condition=" '$(DYNAMOAPI)' == '' ">C:\Program Files\Dynamo 0.7</DYNAMOAPI>
+	<DYNAMOAPI  Condition=" '$(DYNAMOAPI)' == '' ">C:\Program Files\Dynamo\Dynamo Core\1.2</DYNAMOAPI>
```

Comment:
This change is good, except that you need to update the condition to " !Exists('$(DYNAMOAPI)')", then you don't need to change user_local.props

Your current changes to user_local.props will break our automated build system.